package.json

```diff
@@ -104,7 +104,7 @@
     "format": "prettier --write '**/*.js'",
     "test": "npx wdio",
     "predevelop": "yarn clean-cache && yarn tokens:clean-build",
-    "release": "npx lerna version patch --conventional-commits --changelog-preset angular --message 'chore(release): publish %s' --create-release=github",
+    "release": "npx lerna version --conventional-commits --changelog-preset angular --message 'chore(release): publish %s' --create-release=github",
```

Comment:
According to [the documentation](https://github.com/lerna/lerna/tree/main/commands/version#--create-release-type), it seems to me that there is no need for the equal symbol when using the option `--create-release`